HHS/OPRE-OPS
# Ops 1001/task 1375

## What changed

adds checks for who can edit a BLI

## Issue

#1375 

## How to test

1. Try to edit agreement 1 budget lines. // it should not let you
2. Add yourself as project officer or team member
3. repeat step 1. // it should let you

## Screenshots

_If relevant, e.g. for a front-end feature_

## Links

_If relevant, e.g. for a link to a piece of markdown documentation_
